http.go

```diff
@@ -170,6 +174,13 @@ func probeHTTP(target string, w http.ResponseWriter, module Module) (success boo
 			success = true
 		}
 
+		// If the response was gzipped and was transparently uncompressed
+		if resp.Uncompressed {
+			content_compressed = 1
+			if body, err := ioutil.ReadAll(resp.Body); err == nil {
+				content_length = int64(len(body))
```

Comment:
Here's the code: https://github.com/golang/go/blob/release-branch.go1.7/src/net/http/transport.go#L1524

And from the documentation, https://golang.org/pkg/net/http/#Transport:
>         // DisableCompression, if true, prevents the Transport from
>         // requesting compression with an "Accept-Encoding: gzip"
>         // request header when the Request contains no existing
>         // Accept-Encoding value. If the Transport requests gzip on
>         // its own and gets a gzipped response, it's transparently
>         // decoded in the Response.Body. However, if the user
>         // explicitly requested gzip it is not automatically
>         // uncompressed.